pkg/cloud/ibm/assets/deployment.yaml

```diff
@@ -0,0 +1,114 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ibm-cloud-controller-manager
+  namespace: openshift-cloud-controller-manager
+  labels:
+    k8s-app: ibm-cloud-controller-manager
+spec:
+  replicas: 3
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+      maxSurge: 3
+  selector:
+    matchLabels:
+      k8s-app: ibm-cloud-controller-manager
+  template:
+    metadata:
+      labels:
+        k8s-app: ibm-cloud-controller-manager
+    spec:
+      priorityClassName: system-cluster-critical
+      hostNetwork: true
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - topologyKey: "kubernetes.io/hostname"
+            labelSelector:
+              matchLabels:
+                k8s-app: ibm-cloud-controller-manager
+      tolerations:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
+        operator: Exists
+      - effect: NoExecute
+        key: node.kubernetes.io/unreachable
+        operator: Exists
+        tolerationSeconds: 120
+      - effect: NoExecute
+        key: node.kubernetes.io/not-ready
+        operator: Exists
+        tolerationSeconds: 120
+      - effect: NoSchedule
+        key: node.cloudprovider.kubernetes.io/uninitialized
+        operator: Exists
+      - effect: NoSchedule
+        key: node.kubernetes.io/not-ready
+        operator: Exists
+      serviceAccountName: cloud-controller-manager
+      terminationGracePeriodSeconds: 90
+      containers:
+      # Container is required to specify name in order for image substitution
+      # to take place. Cloud-controller-manager is substituted with
+      # real CCM image from the cluster
+      - name: cloud-controller-manager
+        image: quay.io/openshift/origin-ibm-cloud-controller-manager
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          runAsGroup: 0
+          runAsNonRoot: true
+        env:
+          - name: VPCCTL_CLOUD_CONFIG
+            value: /etc/kubernetes/cloud.conf
+        command:
+          - /bin/bash
+          - -c
+          - |
+            #!/bin/bash
+            set -o allexport
+            if [[ -f /etc/kubernetes/apiserver-url.env ]]; then
+              source /etc/kubernetes/apiserver-url.env
+            fi
+            exec /bin/ibm-cloud-controller-manager \
```

Comment:
Please set `--tls-cipher-suites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,TLS_AES_128_GCM_SHA256,TLS_CHACHA20_POLY1305_SHA256,TLS_AES_256_GCM_SHA384` to use TLS 1.3 ciphers.

Reply:
updated